features/view-site-in-lang-i18n-l10n.feature

```diff
@@ -0,0 +1,35 @@
+Feature: As a visitor
+  In order to view the site in my language
+  I need to be able choose either Swedish or English for the site
+
+  PT: https://www.pivotaltracker.com/story/show/133316647
+
+  Background:
+    Given I am Logged out
+
+
+  Scenario: Default language is Swedish
+    Given I am on the "all companies" page
+    Then I should see "Hitta H-märkt företag"
+    And I should not see "Swedish flag"
+    And I should see "English flag"
+    And I should see t("theme_copyright", locale: :sv)
+
+
+  Scenario: Visitor switches the site language from English to Swedish
+    Given I am on the "all companies" page
+    When I click on "change-lang-to-english"
+    When I click on "change-lang-to-svenska"
```

Comment:
I'm thinking the same thing as I'm converting them to use `t('.....')` for localization.  It makes the tests a bit less readable.  I could write steps that hide all of the `t(...)` calls, but that makes the # of steps grow tremendously.  In this situation, it's not worth it at all.  But I could certainly see situations when it'd be necessary.